pkg/http/api.go

```diff
@@ -1807,7 +1814,7 @@ func (a *api) onGetMetadata(reqCtx *fasthttp.RequestCtx) {
 
 	// Copy synchronously so it can be serialized to JSON.
 	a.extendedMetadata.Range(func(key, value interface{}) bool {
-		temp[key.(string)] = key.(string)
+		temp[key.(string)] = value.(string)
```

Comment:
Great catch

Reply:
This is already fixed upstream. I ran into this very problem today with @diegoAJR :)